src/odemis/gui/comp/text.py

```diff
@@ -275,7 +275,7 @@ def SetChoices(self, choices):
         self.dropdownlistbox.SetWindowStyleFlag(flags)
         if not isinstance(choices, list):
             self._choices = list(choices)
-        self._choices.sort(cmp=locale.strcoll)
+        self._choices.sort()
```

Comment:
That used to sort *case insensitive*. It's still useful, so to keep doing the same: `self._choices.sort(key=str.lower)`

Reply:
It seems to me that str.lower and locale.strcoll do opposite things:
```
In [6]: x=['a','C',"B",'t']

In [7]: x.sort(cmp=locale.strcoll)

In [8]: x
Out[8]: ['B', 'C', 'a', 't']

In [9]: x.sort(key=str.lower)

In [10]: x
Out[10]: ['a', 'B', 'C', 't']
```

Reply:
that's in python2

Reply:
`x.sort()` in this case does the same as `x.sort(cmp=locale.strcoll)`

Reply:
On my machine, `strcoll` is case-insensitive. Anyway, the original goal was to be case-insensitive, so let's use `str.lower`.